pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -912,9 +918,14 @@ func (a *Actuator) requestPowerOff(ctx context.Context, baremetalhost *bmh.BareM
 		return &machineapierrors.RequeueAfterError{RequeueAfter: time.Second * 5}
 	}
 
-	baremetalhost.Annotations[requestPowerOffAnnotation] = ""
+	// Issue a hard reboot for immediate remediation purposes
+	remediationRebootAnnotation := bmh.RebootAnnotationArguments{Mode: bmh.RebootModeHard}
+	remediationString, err := json.Marshal(remediationRebootAnnotation)
```

Comment:
We're ignoring any error in marshalling here... it seems like we should at least log it.

Reply:
@zaneb I don't disagree, but I figured it would be safe given that it's our own struct that we're marshalling, not data that we're getting from a client/user.